src/ClientData/include/ClientData/ProcessData.h

```diff
@@ -97,8 +95,6 @@ class ProcessData final {
   mutable absl::Mutex mutex_;
   orbit_grpc_protos::ProcessInfo process_info_;
 
-  // This is a map from module_path to the space in memory where that module is loaded
-  absl::node_hash_map<std::string, ModuleInMemory> module_memory_map_;
   std::map<uint64_t, ModuleInMemory> start_address_to_module_in_memory_;
```

Comment:
Since we are now iterating quite a lot over this map we would probably benefitting from making it a flat_hash_map. As far as I can see we don't need this to be an ordered container, do we?

Reply:
FindModuleByAddress assumes this is sorted by start_address because it uses upper_bound.

Reply:
Ahh, missed that. Thanks.